pennylane/plugins/expt_tensornet.py

```diff
@@ -84,19 +85,25 @@ class TensorNetwork(Device):
         "Identity": identity,
     }
 
+    backend = "numpy"
+    reshape = staticmethod(np.reshape)
+    array = staticmethod(np.array)
+    asarray = staticmethod(np.asarray)
+    real = staticmethod(np.real)
+    imag = staticmethod(np.imag)
+    abs = staticmethod(np.abs)
+
+    C_DTYPE = np.complex128
+    R_DTYPE = np.float64
+
     def __init__(self, wires, shots=1000, analytic=True):
         super().__init__(wires, shots)
-        self.eng = None
         self.analytic = True
         self._nodes = []
         self._edges = []
-        self._zero_state = np.zeros([2] * wires)
-        self._zero_state[tuple([0] * wires)] = 1.0
-        # TODO: since this state is separable, can be more intelligent about not making a dense matrix
-        self._state_node = self._add_node(
-            self._zero_state, wires=tuple(w for w in range(wires)), name="AllZeroState"
-        )
-        self._free_edges = self._state_node.edges[:]  # we need this list to be distinct from self._state_node.edges
+        self._state_node = None
+        self._free_edges = []
+        self.reset()
```

Comment:
The changes made to `expt_tensornet` are solely to make it backend agnostic, to avoid code duplication in `expt_tensornet_tf`.